testData/solutions/win32.json5

```diff
@@ -831,63 +1037,86 @@
                             "default": 1
                         }
                     },
-                    "HTML.WrapNavigation": {
+                    "HTML.FormFieldPromptOptions": {
                         "schema": {
-                            "title": "Wrap Navigation",
-                            "description": "Whether or not to allow navigation keystrokes such as tab and shift tab to wrap to the top or bottom of the document.",
+                            "title": "Form Field Prompt Options",
+                            "description": "Which attributes to prefer when describing form fields.",
                             "enum": [
                                 0,
-                                1
+                                1,
+                                2,
+                                3,
+                                4,
+                                5
                             ],
                             "enumLabels": [
-                                "off",
-                                "on"
+                                "Label Tag",
+                                "Title",
+                                "Alt Text",
+                                "Longest of the above",
+                                "Both Label and Title if different",
+                                "Both Label and Alt if different"
                             ],
-                            "default": 1
+                            "default": 0
                         }
                     },
-                    "Options.AllCapsIndicator": {
+                    "HTML.PageRefreshFilter": {
+                        // NOTE-TODO: Even if the setting is an integer the values
+                        // that is possible to set through the interface are three:
+                        //    - Refresh As Needed: 0
+                        //    - Refresh Every 60 Seconds: 60000
+                        //    - Suppress All Refreshes: -1
                         "schema": {
-                            "title": "AllCapsIndicator",
-                            "description": "The text to read before text that appears in all capital letters.",
-                            "type": "string",
-                            "default": "all cap"
+                            "title": "Page Refresh Filter",
+                            "description": "How often, in milliseconds, to allow all refreshes. Set to 0 to allow all refreshes without any delay.",
+                            "type": "integer",
+                            "default": 0
```

Comment:
Seems reasonable to at least add a minimum of zero here.

Reply:
this has been commented out.